src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -31,13 +29,13 @@ public CinemaHallController(CinemaHallService cinemaHallService,
         this.cinemaHallDtoResponseMapper = cinemaHallDtoResponseMapper;
     }
 
-    @PostMapping
+    @PostMapping("/cinema-halls")
     public CinemaHallResponseDto addCinemaHall(@RequestBody CinemaHallRequestDto dto) {
         CinemaHall cinemaHall = cinemaHallService.add(cinemaHallDtoRequestMapper.fromDto(dto));
         return cinemaHallDtoResponseMapper.toDto(cinemaHall);
     }
 
-    @GetMapping
+    @GetMapping("/cinema-halls")
     public List<CinemaHallResponseDto> getAllCinemaHalls() {
```

Comment:
move back to RequestMapping,
you didnt need RequestMapping in AuthenticationController because in that controller you can add /login endpoint for example
and /register/login is not a correct endpoint name 

<img width="614" alt="image" src="https://user-images.githubusercontent.com/81781479/203858892-b53301ef-2a54-4915-b256-04428f18b3b6.png">